colossus-metrics/src/main/scala/colossus/metrics/collectors/CollectorConfigLoader.scala

```diff
@@ -20,4 +21,10 @@ private[metrics] trait CollectorConfigLoader {
       }
     }
   }
+
```

Comment:
This is an abstraction of that dupe code that was previously in every Collector.